### PR TITLE
sdk(react-native-sdk): fixes

### PR DESCRIPTION
**Pull request by Calinteodor**
image

### CODE REVIEW

**Comment by saghul** [comment]
Better start using __fírmame which contains the directory of the current script and make the paths relative to it.

File: react-native-sdk/update_dependencies.js [hunk 1]
```diff
@@ -1,8 +1,10 @@
 /* eslint-disable guard-for-in */
+/* global __dirname */
 
 const fs = require('fs');
+const path = require('path');
 
-const packageJSON = require('../../package.json');
+const packageJSON = path.resolve(__dirname, '../../package.json');
```
Comment (saghul) [inline react-native-sdk/update_dependencies.js:7]: You are not require-ing this anywhere...

File: react-native-sdk/update_dependencies.js [hunk 2]
```diff
@@ -1,8 +1,10 @@
...
 
 const RNSDKpackageJSON = require('./package.json');
```
Comment (saghul) [inline react-native-sdk/update_dependencies.js:9]: Use surname here as well pl

**Review by saghul — DISMISSED** [review]

File: react-native-sdk/update_dependencies.js [hunk 3]
```diff
@@ -32,14 +44,9 @@ function updateDependencies() {
...
 
-    console.log(
-        'Updating dependencies:',
-        Object.keys(packageJSON.dependencies)
-    );
-
     const data = JSON.stringify(packageJSON, null, 2);
 
-    fs.writeFileSync('../../package.json', data);
+    fs.writeFileSync(packageJSON, data);
```
Comment (saghul) [inline react-native-sdk/update_dependencies.js:49]: This is not a path but an object.

Comment (Calinteodor) [reply react-native-sdk/update_dependencies.js:49]: So, this should be fs.writeFileSync(path.resolve(__dirname, '../../package.json'), data).

**Review by saghul — APPROVED** [review]

**Pull request merged**